README.md

```diff
@@ -23,4 +23,28 @@ yarn global add @aweary/alder
     -e, --exclude <s>  Exclude files matching a pattern
     -i, --include <s>  Include only files that match a pattern
     -d, --depth <n>    Only render the tree to a specific depth
-```
\ No newline at end of file
+```
+
+### Exclude pattern
+
+You can pass either a string or a regular expression as string to `--exclude`:
+
+```sh
+# excluding single directory
+alder --exclude=.git
+
+# excluding multiple directories
+alder --exclude=".git|bower_components|node_modules"
+```
+
+### Include pattern
+
+You can pass either a string or a regular expression as string to `--include`:
```

Comment:
Could you change both of these to say:
> You can pass a string that will be parsed as a regular expression to `--include`

Since alder always just uses `new RegExp`, I don't think we need to make the distinction 👍